Find a functionally equivalent code:com.android.internal.policy.impl.PhoneWindowManager.goHome:COMMENT
Method Modifier: internal    
Comment:/**
 * goes to the home screen
 * @return whether it did anything
 */

@@ -18,30 +18,30 @@
                 sendCloseSystemWindows();
                 Intent dock = createHomeDockIntent();
                 if (dock != null) {
-                    int result = ActivityManagerNative.getDefault().startActivityAsUser(null, null, dock, dock.resolveTypeIfNeeded(mContext.getContentResolver()), null, null, 0, ActivityManager.START_FLAG_ONLY_IF_NEEDED, null, null, null, UserHandle.USER_CURRENT);
+                    int result = ActivityManagerNative.getDefault().startActivityAsUser(null, null, dock, dock.resolveTypeIfNeeded(mContext.getContentResolver()), null, null, 0, ActivityManager.START_FLAG_ONLY_IF_NEEDED, null, null, UserHandle.USER_CURRENT);
                     if (result == ActivityManager.START_RETURN_INTENT_TO_CALLER) {
                         return false;
                     }
                 }
             }
-            int result = ActivityManagerNative.getDefault().startActivityAsUser(null, null, mHomeIntent, mHomeIntent.resolveTypeIfNeeded(mContext.getContentResolver()), null, null, 0, ActivityManager.START_FLAG_ONLY_IF_NEEDED, null, null, null, UserHandle.USER_CURRENT);
+            int result = ActivityManagerNative.getDefault().startActivityAsUser(null, null, mHomeIntent, mHomeIntent.resolveTypeIfNeeded(mContext.getContentResolver()), null, null, 0, ActivityManager.START_FLAG_ONLY_IF_NEEDED, null, null, UserHandle.USER_CURRENT);
             if (result == ActivityManager.START_RETURN_INTENT_TO_CALLER) {
                 return false;
             }

